### PR TITLE
[16.0][MIG] sale_attached_product

### PR DESCRIPTION
**Pull request by cormaza**

### CODE REVIEW

File: sale_attached_product/models/sale_order.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def _update_attached_lines(self):
                 line.attached_line_ids.update({"product_uom_qty": line.product_uom_qty})
                 # Trigger possible pricelist changes
                 for attached_line in line.attached_line_ids:
-                    attached_line.product_uom_change()
+                    attached_line._compute_product_uom_qty()
```
Comment (pedrobaeza) [inline sale_attached_product/models/sale_order.py]: Please call directly `_compute_price_unit`

**Comment by rousseldenis** [comment]
@chienandalu @pedrobaeza Is this a more simple module for product packs ?

File: sale_attached_product/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+This module allows to define a set of products which will be added atomatically to the
```
Comment (pedrobaeza) [inline sale_attached_product/readme/DESCRIPTION.rst]:
```suggestion
This module allows to define a set of products which will be added automatically to the
```

**Comment by pedrobaeza** [comment]
@rousseldenis not really. This is intended for some kind of accessories that are pre-added to the sales order, and then you can remove what you want or change quantities.

**Review by FrancoMaxime — APPROVED** [review]
LGTM: code review

**Review by chienandalu — APPROVED** [review]
Tested :+1: thanks :)

**Pull request merged**